app/models/concerns/transitions_status.rb

```diff
@@ -1,6 +1,9 @@
 module TransitionsStatus
   extend ActiveSupport::Concern
 
+  # Statuses that indicate completion (success, failure, or requiring review)
```

Comment:
Would naming this COMPLETION_STATUSES instead of PROGRESSED_STATUSES be clearer?

Reply:
Sure! Part of the follow up cleanup has:

```diff
+    def completed?
+      PROGRESSED_STATUSES.include?(status)
+    end
```

So `COMPLETION_STATUSES` looks like a better fit 👍